src/contracts/pods/EigenPodPausingConstants.sol

```diff
@@ -18,4 +18,9 @@ abstract contract EigenPodPausingConstants {
     uint8 internal constant PAUSED_EIGENPODS_VERIFY_BALANCE_UPDATE = 3;
     /// @notice Index for flag that pauses the `verifyBeaconChainFullWithdrawal` function *of the EigenPods* when set. see EigenPod code for details.
     uint8 internal constant PAUSED_EIGENPODS_VERIFY_WITHDRAWAL = 4;
+
+    /// @notice Index for flag that pauses deposits to EigenPodManager when set.
+    uint8 internal constant PAUSED_DEPOSITS = 5;
+    /// @notice Index for flag that pauses withdrawals from EigenPodManager when set.
+    uint8 internal constant PAUSED_WITHDRAWALS = 6;
```

Comment:
do we need a new flag here?
It seems like `PAUSED_DEPOSITS` is being used the same way  `PAUSED_EIGENPODS_VERIFY_CREDENTIALS` was designed, and likewise that `PAUSED_WITHDRAWALS` is being used the same way `PAUSED_WITHDRAW_RESTAKED_ETH` was designed. 

also looks like the comments on this file didn't get updated at all.

Reply:
Agreed, fixed!